testcrate/build.rs

```diff
@@ -46,16 +50,18 @@ fn main() {
 
     // These platforms do not have f16 symbols available in their system libraries, so
     // skip related tests. Most of these are missing `f16 <-> f32` conversion routines.
-    if (target.starts_with("aarch64-") && target.contains("linux"))
-        || target.starts_with("arm")
-        || target.starts_with("powerpc-")
-        || target.starts_with("powerpc64-")
-        || target.starts_with("powerpc64le-")
-        || target.starts_with("i586-")
-        || target.contains("windows-")
+    if (target.arch == "aarch64" && target.os == "linux")
+        || target.arch == "arm"
```

Comment:
This isn't the same since there are a few different architectures that start with `arm`, probably best to change it to `target.starts_with("arm")`.

Reply:
Good spot, changed to `target.arch.starts_with("arm")`.